_pages/en/it-picture-long-term.md

```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
 EA is held accountable to:
 
 - Provide an artifact repository which, when used by other teams, puts them into compliance
-- Other teams may freely opt out of their services, and they will be measured against the uptake rate from their clients
```

Comment:
I am putting this one back in ... 
Why are we taking out the one measureable item we have? We talk a lot about metrics, yet seem uncomfortable putting them in. This aligns well with service culture and pushing for the collection of metrics...